tests/Profile.spec.ts

```diff
@@ -350,29 +350,62 @@ test.describe(`Testfälle für das eigene Profil anzeigen: Umgebung: ${process.e
       await expect(profileView.label_KopersNr).toBeHidden();
       await expect(profileView.data_KopersNr).toBeHidden();
       await expect(profileView.icon_InfoPersoenlicheDaten).toBeVisible();
-      // Schulzuordnung 1
-      await expect(profileView.cardHeadline_Schulzuordnung1).toHaveText('Schulzuordnung 1');
-      await expect(profileView.label_Schule1).toHaveText('Schule:');
-      await expect(profileView.data_Schule1).toHaveText(organisation1);
-      await expect(profileView.label_Rolle1).toHaveText('Rolle:');
-      await expect(profileView.data_Rolle1).toHaveText(rollenname);
-      await expect(profileView.label_Dienststellennummer1).toHaveText('DStNr.:');
-      await expect(profileView.data_Dienststellennummer1).toHaveText(dienststellenNr1);
-      // Schulzuordnung 2
-      await expect(profileView.cardHeadline_Schulzuordnung2).toHaveText('Schulzuordnung 2');
-      await expect(profileView.label_Schule2).toHaveText('Schule:');
-      await expect(profileView.data_Schule2).toHaveText(organisation2);
-      await expect(profileView.label_Rolle2).toHaveText('Rolle:');
-      await expect(profileView.data_Rolle2).toHaveText(rollenname);
-      await expect(profileView.label_Dienststellennummer2).toHaveText('DStNr.:');
-      await expect(profileView.data_Dienststellennummer2).toHaveText(dienststellenNr2);
-      // Passwort
-      await expect(profileView.cardHeadline_Passwort).toHaveText('Passwort');
-      await expect(profileView.button_NeuesPasswortSetzen).toBeEnabled();
-      // 2FA
-      // Aktuell wird der Abschnitt 2FA generell nicht angezeigt
-      // await expect(profileView.cardHeadline_2FA).toHaveText('Zwei-Faktor-Authentifizierung');
-      // await expect(profileView.button_2FAEinrichten).toBeEnabled();
+      
+      // prüfen, welche von den beiden Schulen zuerst angezeigt wird in der Tabelle
+      if(await profileView.data_Schule1.innerText() == organisation1) {
```

Comment:
If Condition im Test ist aus meiner Sicht keine gute Idee :)

Reply:
Die beiden Schulzuordnungen kommen in einer zufälligen Reihenfolge rein...
Finde ich auch nicht schön, so ist der Test aber stabil.
Mir fällt keine bessere Lösung ein. Ein change im BE würde das Problem lösen. Das ist jetzt aber nicht der richtige Zeitpunkt.

Reply:
Man könnte die Daten aus dem BE sortieren damit sie in der richtigen Reihenfolge sind aber das ist nicht der richtige Zeitpunkt.